src/streamlink/plugins/bigo.py

```diff
@@ -59,13 +62,23 @@ def can_handle_url(cls, url):
     def _get_streams(self):
         page = http.get(self.url,
                         allow_redirects=True,
-                        headers={"User-Agent": "Mozilla/5.0 (MSIE 10.0; Windows NT 6.1; Trident/5.0)"})
+                        headers={"User-Agent": useragents.CHROME})
         flashvars = self._flashvars_re.search(page.text)
         if not flashvars:
             return
 
         sid, uid, ip, port = flashvars.groups()
-        yield "live", BigoStream(self.session, sid, uid, ip, port)
+        yield "flv", BigoStream(self.session, sid, uid, ip, port)
```

Comment:
Doesn't it work if both streams are called `live`? HLS shout have a higher priority.